_posts/2021-09-29-solidity-0.8.9-release-announcement.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.9 Release Announcement'
+date: '2021-09-29'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.9](https://github.com/ethereum/solidity/releases/tag/v0.8.9) fixes two important bugs:
```

Comment:
```suggestion
[Solidity v0.8.9](https://github.com/ethereum/solidity/releases/tag/v0.8.9) is a pure bugfix release and fixes two important, but low severity, bugs:
```

Q: Are the bugs "important" when they are both categorized as "very low" in severity?